internal/static/integrations/mysqld_exporter/mysqld-exporter.go

```diff
@@ -122,7 +123,7 @@ func New(log log.Logger, c *Config) (integrations.Integration, error) {
 	}
 
 	scrapers := GetScrapers(c)
-	exporter := collector.New(context.Background(), string(dsn), scrapers, log, collector.Config{
+	exporter := collector.New(context.Background(), string(dsn), scrapers, slog.Default(), collector.Config{
```

Comment:
@thampiotr `New` here now requires an `slog.Logger`, is there any adapter in the Alloy codebase?